Makefile

```diff
@@ -167,3 +167,15 @@ release:
 	@make release-kubectl-karmada GOOS=linux GOARCH=arm64
 	@make release-kubectl-karmada GOOS=darwin GOARCH=amd64
 	@make release-kubectl-karmada GOOS=darwin GOARCH=arm64
+
+.PHONY: install-addlicense
+## install-addlicense: check license if not exist install addlicense tools
+install-addlicense:
+ifeq (, $(shell which addlicense))
+	go install github.com/zhy76/addlicense@latest
```

Comment:
I have researched and try many available license tools, and either some of the comment formats do not meet the requirements, or the year cannot be specified, or the supported file types are limited. After comparison, I think Google's tool(https://github.com/google/addlicense) is the best option for our situation, but its comment format for go files does not meet our requirements and it has no updated for months, so I fork it and only modify the comment format of go files: https://github.com/zhy76/addlicense/blob/e1d9e46e60f9f6e9964092f07d6c2c2e83a77655/main.go#L292.

Or I can delete the update-license script but keep the verify-license script. Users can add comments manually in the future and ci will verify the license.

The current invocation of update-license.sh looks like this:
<img width="815" alt="image" src="https://github.com/karmada-io/karmada/assets/56665618/3489a8ba-71a2-4f70-8bbd-55ff65fa64bc">
The script searches for the target files and automatically adds license if the file has no license, and it will prompt message：xxx is modified.

The current invocation of verify-license.sh looks like this:
<img width="622" alt="image" src="https://github.com/karmada-io/karmada/assets/56665618/74feb895-43c9-4d5b-867f-8199906d9bac">

<img width="660" alt="image" src="https://github.com/karmada-io/karmada/assets/56665618/df5c810f-dd79-407d-b5eb-c4e29d819de2">

Reply:
> Or I can delete the update-license script but keep the verify-license script. Users can add comments manually in the future and ci will verify the license.

Make sense. With the verification, we are unlikely to miss the license any more.

Reply:
OK, I deleted the update-license.sh.